# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/stf/avview/gcc34.patch
# ROCK Linux is Copyright (C) 1998 - 2005 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -dur avview-0.80.3/alsa.c avview-0.80.3-patch/alsa.c
--- avview-0.80.3/alsa.c	2004-01-03 18:17:58.000000000 +0000
+++ avview-0.80.3-patch/alsa.c	2004-07-12 23:32:34.000000000 +0000
@@ -297,7 +297,6 @@
 				}
 			Tcl_ListObjAppendElement(interp,list,list2);
 			break;
-		default:
 		}
 	Tcl_ListObjAppendElement(interp, ans, list);
 	j++;
Nur in avview-0.80.3-patch: alsa.c~.
diff -dur avview-0.80.3/vbi.c avview-0.80.3-patch/vbi.c
--- avview-0.80.3/vbi.c	2004-01-03 18:18:03.000000000 +0000
+++ avview-0.80.3-patch/vbi.c	2004-07-15 23:37:56.000000000 +0000
@@ -98,10 +98,10 @@
 				Tcl_VarEval(data->interp, data->event_command, " aspect {}", NULL);
 				break;
 			
-			default:
+
 			}
 		break;
-	default:
+
 	}
 }
 
